setup.py

```diff
@@ -11,6 +11,7 @@
     packages=find_packages(exclude=["test_project", ]),
     install_requires=[
         "django<2.0",
+        "psycopg2",
```

Comment:
If a requirement exists in setup.py, then it doesn't need to exist in the requirements.txt file used for testing. In this case, I think `psycopg2` should be removed from the setup.py requirements.